src/pages/Events/index.jsx

```diff
@@ -25,7 +25,7 @@ const Events = () => {
   }
 
   if (isError) {
-    return <components.Error />;
+    return <components.Loading />;
```

Comment:
revert this please and I only mentioned that you should add a loading component.